### PR TITLE
Per Issue #163 - Indicate support for Web Authentication

### PR DESCRIPTION
**Pull request by jcjones**
Not sure how much to write in the detail section, but here's a first pass, per Issue #163.

### CODE REVIEW

File: activities.json [hunk 1]
```diff
@@ -219,6 +219,17 @@
     "title": "Transport Layer Security (TLS) Certificate Compression",
     "url": "https://tools.ietf.org/html/draft-ietf-tls-certificate-compression"
   },
+  {
+    "ciuName": null,
```
Comment (dbaron) [inline activities.json]: This should be "webauthn" to point to https://caniuse.com/#feat=webauthn

File: activities.json [hunk 2]
```diff
@@ -219,6 +219,17 @@
...
+    "description": "This specification defines an API enabling the creation and use of strong, attested, scoped, public key-based credentials by web applications, for the purpose of strongly authenticating users. Conceptually, one or more public key credentials, each scoped to a given WebAuthn Relying Party, are created by and bound to authenticators as requested by the web application. The user agent mediates access to authenticators and their public key credentials in order to preserve user privacy. Authenticators are responsible for ensuring that no operation is performed without user consent. Authenticators provide cryptographic proof of their properties to Relying Parties via attestation. This specification also describes the functional model for WebAuthn conformant authenticators, including their signature and attestation functionality.",
+    "mozBugUrl": null,
```
Comment (dbaron) [inline activities.json]: This should probably be "https://bugzilla.mozilla.org/show_bug.cgi?id=1294514"

File: activities.json [hunk 3]
```diff
@@ -219,6 +219,17 @@
...
+    "mozPosition": "important",
+    "mozPositionDetail": "Public key cryptographic authentication is a major improvement in the fight against phishing, and we encourage all security-concious web applications to implement and innovate authentication flows around Web Authentication in the future.",
```
Comment (dbaron) [inline activities.json]: The use of "innovate" here is a bit awkward since innovate is an intransitive verb and you're giving it a direct object...

Comment (martinthomson) [reply activities.json]: Also typo conscious

Comment (jcjones) [reply activities.json]: Aha, oops. Thanks!

**Pull request merged**